fp/_mapping.js

```diff
@@ -213,7 +217,8 @@ exports.remap = {
   'getOr': 'get',
   'trimChars': 'trim',
   'trimCharsEnd': 'trimEnd',
-  'trimCharsStart': 'trimStart'
+  'trimCharsStart': 'trimStart',
+  'updatePath': 'update'
```

Comment:
actually scratch that. Lets not add an alias unless its warranted for cross-lib migration.